Documentation/install/upgrade.rst

```diff
@@ -309,22 +309,39 @@ IMPORTANT: Changes required before upgrading to 1.7.0
 * Cilium has bumped the minimal kubernetes version supported to v1.11.0.
 
 * The ``kubernetes.io/cluster-service`` label has been removed from the Cilium
-  `DaemonSet` selector. Existing users must remove this label from their
-  `DaemonSet` specification to safely upgrade. If action is not taken to make
-  the selector labels consistent with the upgraded YAMLs, then the new
-  `DaemonSet` YAML which is created during upgrade will fail to apply.
+  `DaemonSet` selector. Existing users must either choose to keep this label on
+  `DaemonSet` specification to safely upgrade or re-create the Cilium `DaemonSet`
+  without the deprecated label. It is advisable to run the with this flag when
+  doing an upgrade from ``v1.6.x`` to ``v1.7.x`` in the event of having to do a
+  downgrade. Removing this label should be done within same Cilium versions.
```

Comment:
Maybe "Removing the label should be done after the upgrade"?

Reply:
Change to `The removal of this label should be done after a successful upgrade.`